proxy-attestation-server/src/cli.rs

```diff
@@ -0,0 +1,118 @@
+//! Proxy Attestation Server command-line interface
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE.markdown` file in the Veracruz root directory for
+//! information on licensing and copyright.
+
+use actix_rt;
+use env_logger;
+use log::{info, error};
+use proxy_attestation_server;
+use std::{env, fs, path, process};
+use structopt::StructOpt;
+use veracruz_utils::policy::{
+    error::PolicyError,
+    policy::Policy,
+};
+
+
+#[derive(Debug, StructOpt)]
+#[structopt(rename_all="kebab")]
+struct Opt {
+    /// Path to policy file
+    #[structopt(parse(from_os_str))]
+    policy_path: path::PathBuf,
+
+    /// Path to CA certificate
+    #[structopt(long, parse(from_os_str))]
+    ca_cert: path::PathBuf,
+
+    /// Path to CA private key
+    #[structopt(long, parse(from_os_str))]
+    ca_key: path::PathBuf,
+
+    /// URL or path to database, may also be provided through the
+    /// DATABASE_URL environment variable
+    #[structopt(long)]
+    database_url: Option<String>,
+
+    /// Enable/disable debugging
+    #[structopt(long)]
+    debug: bool,
+}
+
+
+/// Entry point
+fn main() {
+    // parse args
+    let opt = Opt::from_args();
+
+    // setup logger
+    env_logger::from_env(
+        env_logger::Env::default().default_filter_or("info")
+    ).init();
```

Comment:
You can remove this internal setting on rust_log, but relies on the caller to set up rust_log variable.

Reply:
Ok can do, let me check what the output is then, this program may print nothing. I'll add a "Server running on blablabla" message in that case.

Reply:
Updated 👍, this does reduce the noise in the getting started guide quite a bit